src/main/java/li/cil/tis3d/client/gui/ManualGui.java

```diff
@@ -146,7 +145,7 @@ public void render(final MatrixStack matrices, final int mouseX, final int mouse
         }
 
         if (canScroll() && (isCoordinateOverScrollBar(mouseX - guiLeft, mouseY - guiTop) || isDragging)) {
-            renderTooltip(matrices, StringRenderable.plain(100 * offset() / maxOffset() + "%"), guiLeft + SCROLL_POS_X + SCROLL_WIDTH, scrollButton.y + scrollButton.getHeight() + 1);
+            renderTooltip(matrices, new LiteralText(100 * offset() / maxOffset() + "%"), guiLeft + SCROLL_POS_X + SCROLL_WIDTH, scrollButton.y + scrollButton.getHeight() + 1);
```

Comment:
Why not also `Text::of` here? Is one of the two the preferred way now? Is there a contextual difference I'm missing? Not a blocker, just curious. If there's no reason not to keep this coherent would nice to have.

Reply:
I've personally not read enough vanilla to know which is preferred. The implementation of `Text.of` is this one-liner:
```
return (Text)(string != null ? new LiteralText(string) : LiteralText.EMPTY);
```
and since I can tell from the expression used in TIS-3D here that the passed string will never be null, it felt more straightforward to use `LiteralText` directly. But your coherency argument makes a lot of sense (I didn't have this context in mind when updating the code). I'll append a fixup commit that you can squash via github.